data/AddonsManagerMod.js

```diff
@@ -121,22 +117,59 @@ ACRController.makeButtonUI = function(addonReport)
         label.setAttribute("value", "Compatibility Problems"); // TODO l10n
         hbox.appendChild(label);
 
-        return hbox;
+        hbox_outer.appendChild(hbox);
+    } else {
+        var button = document.createElement("button");
+        button.setAttribute("label", "Report Issue");
+        button.setAttribute("class", "anon-control");
+
+        //button.addEventListener("click", function() { ACRController.openSendReportDialog(addonReport); }, true);
+        button.addEventListener("click", function() {
+            //ACRController.openSendReportDialog(addonReport);
+            self.port.emit("acr_open_submit_report_dialog", addonReport);
+        }, true);
+        hbox_outer.appendChild(button);
     }
+    return hbox_outer;
+}
 
-    var button = document.createElement("button");
-    button.setAttribute("label", "Report Issue"); // TODO l10n
-    button.setAttribute("class", "anon-control");
-    button.setAttribute("owner", "acr");
+ACRController.makeE10sInfo = function(addonReport) {
+    var hbox = document.createElement("hbox");
```

Comment:
Let's not do this. This is a XUL document and jQuery breaks in some cases, believe me I tried...